recipe/install-gcc.sh

```diff
@@ -230,8 +230,25 @@ pushd ${PREFIX}/lib
 ln -sf libgomp.so.${libgomp_ver} libgomp.so
 popd
 
-pushd ${PREFIX}/${CHOST}/sysroot/lib
-ln -sf ../../../lib/libgomp.so libgomp.so
-popd
+# make links to libs in the sysroot
+symtargets=$(find ${PREFIX}/lib -name "libstdc++*.so*")
+for symtarget in ${symtargets}; do
+  symtargetname=$(basename ${symtarget})
+  ln -s ${PREFIX}/lib/${symtargetname} ${PREFIX}/${CHOST}/sysroot/lib/${symtargetname}
+done
```

Comment:
Since we need this only for building, let's just have the symlink only for the `.so` and not `.so.*`
```suggestion
ln -s ${PREFIX}/lib/libstdc++.so ${PREFIX}/${CHOST}/sysroot/lib/libstdc++.so
```